keadm/cmd/keadm/app/cmd/edge/join_others.go

```diff
@@ -121,7 +125,20 @@ func createEdgeConfigFiles(opt *common.JoinOptions) error {
 		edgeCoreConfig = v1alpha2.NewDefaultEdgeCoreConfig()
 	}
 
-	edgeCoreConfig.Modules.EdgeHub.WebSocket.Server = opt.CloudCoreIPPort
+	edgeCoreConfig.Modules.EdgeHub.Quic.Server = net.JoinHostPort(opt.CloudCoreIP, strconv.Itoa(constants.DefaultQuicPort))
```

Comment:
Move L169 of code `host, _, err := net.SplitHostPort(opt.CloudCoreIPPort)` over here and change `opt.CloudCoreIP` to `host`

Reply:
ok，I have made it.